cfgov/v1/migrations/0090_auto_20160609_1603.py

```diff
@@ -0,0 +1,143 @@
+        ),
+        migrations.RunPython(convert_datetimes),
+        migrations.AlterField(
+            model_name='eventpage',
+            name='agenda_items',
+            field=wagtail.wagtailcore.fields.StreamField([(b'item', wagtail.wagtailcore.blocks.StructBlock([(b'start_time', wagtail.wagtailcore.blocks.TimeBlock(required=False, label=b'Start')), (b'end_time', wagtail.wagtailcore.blocks.TimeBlock(required=False, label=b'End')), (b'description', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=False)), (b'location', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=False)), (b'speakers', wagtail.wagtailcore.blocks.ListBlock(wagtail.wagtailcore.blocks.StructBlock([(b'name', wagtail.wagtailcore.blocks.CharBlock(required=False)), (b'url', wagtail.wagtailcore.blocks.URLBlock(required=False))], required=False, icon=b'user')))]))], blank=True),
+        ),
+    ]
```

Comment:
Well `approved_go_live_at` and `go_live_at` are the same date just different fields. There's talk of removing the latter.